### PR TITLE
Add topic about GPL

### PR DESCRIPTION
**Pull request by R4356th**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [-] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [-] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [-] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Curating a new topic or collection

- [-] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [-] My folder contains a `*.png` image (if applicable) and `index.md`
- [-] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> GPL is a widely used freeware license and this topic is used by many repositories.

### CODE REVIEW

**Comment by R4356th** [comment]
How do I compress my image?

**Comment by MikeMcQuaid** [comment]
@R4356th It needs resized in an image editor of your choice.

File: topics/gpl/index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+aliases: gplv2, gplv3, gnu-public-license, gnu-general-public-license
+display_name: GNU General Public License
+short_description: GNU General Public License is a 
+ free software license published by the Free Software Foundation.
```
Comment (MikeMcQuaid) [inline topics/gpl/index.md:5]:
```suggestion
short_description: GNU General Public License is a free software license published by the Free Software Foundation.
```

**Comment by R4356th** [comment]
@MikeMcQuaid, I could not fix the logo issue so I am removing it.

File: topics/gpl/index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+aliases: gplv2, gplv3, gnu-public-license, gnu-general-public-license
+display_name: GNU General Public License
+short_description: GNU General Public License is a free software license published by the Free Software Foundation.
+topic: gpl
+wikipedia_url: https://en.wikipedia.org/wiki/GNU_General_Public_License
+---
+GNU General Public License is a free software license published by the Free Software Foundation whose main terms include making the source code open source and ensures no warranty.
```
Comment (MikeMcQuaid) [inline topics/gpl/index.md:8]:
```suggestion
GNU General Public License is a free software license published by the Free Software Foundation whose main terms guarantee users the freedom to run, study, share and modify the software.
```

**Comment by MikeMcQuaid** [comment]
Thanks again @R4356th!

**Pull request merged**